gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystemJavaStorageClientTest.java

```diff
@@ -0,0 +1,30 @@
+package com.google.cloud.hadoop.gcsio;
+
+import com.google.cloud.hadoop.gcsio.GoogleCloudStorageFileSystemOptions.ClientType;
+import java.io.IOException;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class GoogleCloudStorageFileSystemJavaStorageClientTest
+    extends GoogleCloudStorageFileSystemTestBase {
+
+  @Before
+  public void before() throws Exception {
+    storageClientType = ClientType.STORAGE_CLIENT;
+    super.before();
+  }
+
+  @Override
+  @Ignore("DirectPath is not supported with null credentials")
```

Comment:
We can have CFE path also for Javastorage client, correct?

Reply:
No. JavaStorage is used specifically for gRPC over directPath and we don't have usecase of gRPC over CFE.